src/components/browse/components/AboveSearchTablePanel.js

```diff
@@ -2,180 +2,44 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import url from 'url';
 import _ from 'underscore';
 import { patchedConsoleInstance as console } from './../../util/patched-console';
-import { load } from './../../util/ajax';
-import { getAbstractTypeForType } from './../../util/schema-transforms';
+import { BasicStaticSectionBody } from './../../static-pages/BasicStaticSectionBody';
 
-/** TODO IMPROVE */
 
-class AboveSearchTablePanelStaticContentPane extends React.Component {
+export const AboveSearchTablePanel = React.memo(function AboveSearchTablePanel({ context, placeholderReplacementFxn }) {
```

Comment:
(Very small change / almost irrelevant / unsure) - 

Moved the `React.memo` call up one level to `AboveSearchTablePanel` instead of `SearchHeaderSection`.
Reasoning for this is AboveSearchTablePanel receives 2 props, which inherit from Object (and thus the variables are object references), and thus the memo comparison operation (propA !== pastPropA || propB !== pastPropB) should be a lot faster than the comparison in `SearchHeaderSection` because `SearchHeaderSection` compares strings (lists of characters) and presumably might take a bit longer. Idk 🤷‍♂.